package/harvester-os/files/system/oem/90_network.yaml

```diff
@@ -1,7 +1,27 @@
-name: "Bring up interface"
+name: "Network configuration"
 stages:
+  rootfs:
+    - name: Create DHCP config for boot interfaces
+      if: cat /proc/cmdline | grep -q "harvester.install.automatic=true"
+      commands:
+      - |
+        if [ ! -e /tmp/net.ifaces ]; then
+          echo "net.ifaces is not found. Skip configuring boot interface."
+          exit 0
+        fi
+        
+        for iface in $(</tmp/net.ifaces); do
+          echo "Create DHCP config for $iface"
+        
+          cat > /sysroot/etc/sysconfig/network/ifcfg-$iface <<'EOF'
+        BOOTPROTO='dhcp'
```

Comment:
In PXE boot installation, Do we always get the IP through DHCP and not support static mode?

Reply:
Yeah, that's what we have in the document so far.
PXE boot is supposed to work within a DHCP environment.